pubspec.yaml

```diff
@@ -29,7 +29,8 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-
+  flutter_localizations: # Add this line
```

Comment:
refer to our confluence how we use localization

Reply:
already remove